config.xml

```diff
@@ -3,7 +3,7 @@
     <name>iFarm</name>
     <description>An awesome Ionic/Cordova app.</description>
     <author email="hi@ionicframework" href="http://ionicframework.com/">Ionic Framework Team</author>
-    <content original-src="index.html" src="http://192.168.1.103:8100" />
+    <content src="index.html" />
```

Comment:
what is  the deal with this commit?

Reply:
Don't know, I do nothing, auto changed.

Reply:
its better not to commit what you didnt code on purpose ..

Reply:
but leave it for now. but next time look over every code change during a commit and only commit stuff that your purposefully put there